packages/compiler/src/experimental/index.ts

```diff
@@ -1 +1,12 @@
+export * from "./mutators.js";
+export * from "./realm.js";
 export { unsafe_useStateMap, unsafe_useStateSet } from "./state-accessor.js";
+
+import { createConsoleSink } from "../core/logger/console-sink.js";
+import { createLogger } from "../core/logger/logger.js";
+
+const logSink = createConsoleSink({ pretty: true });
+const logger = createLogger({ sink: logSink, level: "warning" });
+logger.warn(
+  "Functionality from @typespec/compiler/experimental is highly likely to change or be removed.",
+);
```

Comment:
ok so we added another api for the stateMap we want to try to use in the other libraries but this will now make it log for every body so this might be a bit too agressive

Reply:
Don't we want to be super clear to anyone using these apis about what they are getting into?

Reply:
yeah it more that it bleeds to the consumer not just the library author

Reply:
like for example for those apis we added, I moved in experimental because we need to try to use them in our own package but if we decide to change the structure for our own package its not a problem we update all in the same PR.